sdk-src/src/main/java/com/hpe/adm/nga/sdk/Octane.java

```diff
@@ -392,4 +408,13 @@ public String getOctaneClassFactoryClassName() {
             return octaneClassFactoryClassName;
         }
     }
+
+    public static class OctaneCustomSettings {
+//        public static interface OctaneCustomSettings {
+//        public int readTimeout = 6000;
+//        public int connectionTimeout();
+        public int readTimeout = 60000;
+        public int connectionTimeout = 15000;
+
```

Comment:
I am not sure I like this.  I would perhaps prefer to have enums or static values as opposed to public properties.

Reply:
static or enum values would make it impossible to pass different values for different Octane instances(conceivable i think)
I intended to have the simplest, record-like syntax possible.